platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/EventStreamRepairCommand.java

```diff
@@ -63,14 +71,19 @@ public Integer call() throws Exception {
         setupConstructableRegistry();
         final EventStreamSingleFileRepairer repairer = new EventStreamSingleFileRepairer(eventStreamFile);
         if (repairer.repair()) {
-            System.out.println("Event Stream Repaired.");
-            System.out.println("Damaged file: " + eventStreamFile.getAbsolutePath() + DAMAGED_SUFFIX);
-            System.out.println("Repaired file: " + eventStreamFile.getAbsolutePath());
+            logger.info(LogMarker.CLI.getMarker(), "Event Stream Repaired.");
```

Comment:
I think these should be combined into a single multiline log message

Also, is there a reason that you chose to make one branch an `info` log, and the other  a `warn` log?